samples/tfm_integration/tfm_regression_test/src/main.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+
+/* Run the PSA test suite */
+void psa_test(void);
+
+__attribute__((noreturn))
+void main(void)
+{
+	printk("Should not be printed, ensure TF-M NS application is flashed to the device.\n");
```

Comment:
Wording suggestion. Added panic as well so it fails a bit more loudly.
```suggestion
	printk("Should not be printed, expected TF-M's NS application to be run instead.\n");
	k_panic();
```